src/modules/geospatial/dto/get-address-by-postcode-query.dto.ts

```diff
@@ -0,0 +1,16 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { GEOSPATIAL } from '@ukef/constants';
+import { Matches, MaxLength, MinLength } from 'class-validator';
+
+const UK_POSTCODE = /^[A-Za-z]{1,2}[\dRr][\dA-Za-z]?\s?\d[ABD-HJLNP-UW-Zabd-hjlnp-uw-z]{2}$/;
+
+export class GetAddressByPostcodeQueryDto {
+  @ApiProperty({
+    example: GEOSPATIAL.EXAMPLES.POSTCODE,
+    description: 'Postcode to search for',
+  })
```

Comment:
```suggestion
  @ApiProperty({
    example: GEOSPATIAL.EXAMPLES.POSTCODE,
    description: 'Postcode to search for',
    minLength: 5,
    maxLength: 8,
    pattern: UK_POSTCODE.source,
  })
```

Reply:
`UK_POSTCODE.source` returns the RegExp as a string

Reply:
Thanks, changed.